app/cdap/components/GridTable/index.tsx

```diff
@@ -253,6 +266,17 @@ export default function GridTable() {
             })}
         </TableBody>
       </Table>
+      {transformationFunction && (
+        <AddTransformation
+          functionName={transformationFunction}
+          directiveFunctionSupportedDataType={transformationFunctionSupportedDataType}
+          columnData={headerNamesList}
+          missingDataList={dataQuality}
+          callBack={() => {
```

Comment:
plz check if we can manage the props better
also, rename the props according to usecase plz

also, transformationFunction seems to be general term, but cannot relate it to add transformation step panel,
plz rename it and make sure it give proper meaning to what we are trying to achieve

transformationFunctionSupportedDataType
this can be just supported data types
or we can keep the supported data types in the transformation function state itself, as each function may have its own supported data types, plz let me know if this makes sense and update the code accordingly